examples/04-grav/plot_laguna_del_maule_inversion.py

```diff
@@ -24,15 +25,16 @@
 def run(plotIt=True, cleanAfterRun=True):
 
     # Start by downloading files from the remote repository
-    url = "https://storage.googleapis.com/simpeg/Chile_GRAV_4_Miller/"
-    cloudfiles = [
-        'LdM_grav_obs.grv', 'LdM_mesh.mesh',
-        'LdM_topo.topo', 'LdM_input_file.inp'
-    ]
-
-    # Download to Downloads/SimPEGtemp
-    basePath = os.path.expanduser('~/Downloads/simpegtemp')
-    download([url+f for f in cloudfiles], folder=basePath, overwrite=True)
+    # directory where the downloaded files are
+
+    url = "https://storage.googleapis.com/simpeg/Chile_GRAV_4_Miller/Chile_GRAV_4_Miller.tar.gz"
+    downloads = download(url, overwrite=True)
+    basePath = downloads.split(".")[0]
```

Comment:
I have also downloaded to the current directory (a folder called Chile_GRAV_4_Miller will be created). This makes it easier to work with different systems and to keep track of the files (esp on travis)